rb/spec/unit/selenium/webdriver/chrome/driver_spec.rb

```diff
@@ -111,6 +110,17 @@ module Chrome
           Driver.new(http_client: http, desired_capabilities: custom_caps, args: %w[baz])
         end
 
+        it 'does not merge empty options' do
```

Comment:
This actually isn't testing this if you check your code. Here you are passing in 2 arguments and verifying you receive 2 args. This is still a missing use case. But then you should have the blank one.

Reply:
Thank you for feedback.
I don't understand this one.
You mean check the case when `goog:chromeOptions` is empty in args in custom desired_capabilities?

Reply:
Also I tried to refactor specs to be more DRY, lmk if it worths it